frame/staking/src/tests.rs

```diff
@@ -2714,14 +2715,23 @@ mod offchain_phragmen {
 		bond_nominator(voter, 1000 + voter, 100, vec![21, 31, 41]);
 	}
 
-	fn offchainify(ext: &mut TestExternalities) -> Arc<RwLock<PoolState>> {
-		let (offchain, _state) = TestOffchainExt::new();
-		let (pool, state) = TestTransactionPoolExt::new();
+	/// convert an externalities to one that can handle offchain worker tests.
+	fn offchainify(ext: &mut TestExternalities, iterations: u32) -> Arc<RwLock<PoolState>> {
+		let (offchain, offchain_state) = TestOffchainExt::new();
+		let (pool, pool_state) = TestTransactionPoolExt::new();
+
+		let mut encoded = iterations.encode();
+		encoded.resize(32, 0u8);
+		let seed: [u8; 32] = encoded
+			.as_slice()
+			.try_into()
+			.expect("lengths are equal; cannot panic; qed");
+		offchain_state.write().seed = seed;
```

Comment:
@tomusdrw  suggestion:
```suggestion
	    let mut seed = [0_u8; 32];
	    seed[0..4].copy_from_slice(&iterations.to_le_bytes());
	    offchain_state.write().seed = seed;
```